.github/workflows/sync-main-to-preprod-and-prod.yml

```diff
@@ -102,6 +90,22 @@ jobs:
           # TODO this does not check whether the updates are in main or in prod, only that files differ. We could
           # make this more exact later to check that the updates are in main (anything else is unexpected).
           diff -qr repository/repository sigstore-tuf-root | grep -Po '([0-9\.]*\w+[\.\w+]*(?= differ))|((Only in \w+\: )\K(.*))' | while read l; do check_filename $l; done
+      - name: sync
+        id: sync
+        if: steps.diff-main-prod.outputs.abort != 'true'
```

Comment:
Given these happen in the same job, you can use  `${{ env.ABORT != 'false' }}` and remove the output from the job which would be unused.